src/Core/Header.php

```diff
@@ -94,7 +94,7 @@ public static function setupHeader($assets = [], $echoOutput = true)
      * @param boolean $headerMode - if true, then include autoloaded assets
      *                              if false, then do not include autoloaded assets
```

Comment:
place the comments to the new parameter here; basically just copy/paste what is in above setupHeader comments

Reply:
How can I forgot about the documentation, I guess I need some sleep(. ❛ ᴗ ❛.).